esbuild.js

```diff
@@ -1,13 +1,21 @@
 import esbuild from 'esbuild';
 import { sassPlugin } from 'esbuild-sass-plugin';
 
-const context = await esbuild.context({
-	entryPoints: ['frontend/src/Index.tsx', 'frontend/static/scss/App.scss'],
-	outdir: 'public/assets',
+await esbuild.build({
+	entryPoints: ['public/scss/custom.scss'],
+	outdir: 'public/css',
 	bundle: true,
 	minify: true,
 	plugins: [sassPlugin()],
 });
 
+// const context = await esbuild.context({
+// 	entryPoints: ['public/scss/custom.scss'],
+// 	outdir: 'public/css',
+// 	bundle: true,
+// 	minify: true,
+// 	plugins: [sassPlugin()],
+// });
+
```

Comment:
what needs to be done here? is it getting removed?

Reply:
removed